db/migrate/20150203165922_create_followings.rb

```diff
@@ -0,0 +1,8 @@
+class CreateFollowings < ActiveRecord::Migration
+  def change
+    create_table :followings do |t|
+      t.timestamps null:false
```

Comment:
`null: false` is not necessary because Rails creates the data with the timestamps. It will not make it into your db without timestamps.